modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/graphql/data/processor/LiferayMethodDataFetchingProcessor.java

```diff
@@ -106,6 +108,11 @@ public Object process(
 			Object source)
 		throws Exception {
 
+		Pagination pagination = _paginationProvider.getPagination(
+			_portal.getCompanyId(httpServletRequest),
+			GetterUtil.getInteger(arguments.get("page"), 1),
```

Comment:
The default value is being duplicated here and in the PaginationContextProvider. Maybe the best option would be accepting null integer values in page and pageSize in the PaginationProvider (by using Integer classes) and then, inside, calculating proper values based on the default values. With that, we would be centralizing the place where the default values are set